cl/favorites/utils.py

```diff
@@ -51,7 +52,13 @@ async def create_prayer(
         new_prayer, created = await Prayer.objects.aget_or_create(
             user=user, recap_document=recap_document
         )
-        return new_prayer if created else None
+        if created:
+            if not fetch_prayer_availability.delay(recap_document.pk):
```

Comment:
This wont work as expected. The delay method returns immediately upon invocation, without awaiting the completion of the associated task and the returned value is an [AsyncResult](http://docs.celeryproject.org/en/latest/reference/celery.result.html#celery.result.AsyncResult%20AsyncResult) instance.